.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+### Description
+What does this PR do?
+
+### TODO only if you're a HashiCorp employee
+- [ ] **Changelog:** Make sure there is one, or the `pr/no-changelog` label is
+  applied. Brand new features have a differently
+  formatted changelog than other changelogs, so pay attention to that. If this
+  PR is the CE portion of an ENT PR, put the changelog here, _not_ in your ENT
+  PR. Feel free to refer to the [instructions](https://github.com/hashicorp/vault/blob/main/CONTRIBUTING.md#changelog-entries) on changelog formatting if necessary.
+- [ ] **Labels:** If this PR is the CE portion of an ENT change, and that ENT change is
+  getting backported to N-2, use the new style `backport/ent/x.x.x+ent` labels
+  instead of the old style `backport/x.x.x` labels.
```

Comment:
```suggestion
- [ ] **Labels:** If this PR is part of a change is
  getting backported to N-2, use the `backport/ent/x.x.x+ent` style labels
  instead of the `backport/x.x.x` style labels.
```